modules/trusted-firmware-m/Kconfig

```diff
@@ -469,7 +469,7 @@ config TFM_NRF_PROVISIONING
 	help
 	  Provision the TF-M image. When enabled, TF-M must be in the PSA
 	  provisioning lifecycle state in order to boot. See provisioning
-	  guide for more details.
+	  in Running applications with Trusted Firmware-M for more details.
```

Comment:
Find more information about provisioning in Running applications with Trusted Firmware-M.